src/components/CancelBlock.tsx

```diff
@@ -1,5 +1,9 @@
 import React, { FunctionComponent, useState } from "react";
-import { TextInput, PrimaryButton, Spinner, Logger, ConfirmationModalDialog, SecondaryButton } from "./common";
+import { TextInput } from "./common/TextInput";
```

Comment:
you can eventually create a `./common/index` that exports all the component

When I say `eventually` it means that whatever decision you make I will approve it :)

Reply:
Note that can merge directly if you don't do any changed